src/pages/LocationsPage.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Location from '../components/Location'
+import data from '../data.json'
+
+LocationList.propTypes = {
+  locations: PropTypes.arrayOf(
+    PropTypes.shape({
+      image: PropTypes.string,
+      description: PropTypes.string,
+    })
```

Comment:
👌🏼

Reply:
What do you mean with "missing on id and image in the `PropTypes.shape`"?

Reply:
So the `locations` is an array containing objects. And you won't need the id.
One object contains that's what Anni means, I think:
```js
PropTypes.shape({
  image: PropTypes.string,
  description: PropTypes.string,
  name: PropTypes.string
})
```

Reply:
However, your component `LocationList` doesn't get any props so you could delete them. Or import your `data` in the `App.js` and pass it as props (might be best because later you'll want one entry point of your data which typically is the `App.js`).